### PR TITLE
Do not initialize install_yamls in architecture scenario

### PR DESCRIPTION
**Pull request by cjeanner**
As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running
  - [X] @cjeanner tested with HCI deployment locally
  - [X] downstream job essentially passed (failure hit, but on non-related step)

### CODE REVIEW

File: playbooks/01-bootstrap.yml [hunk 1]
```diff
@@ -44,10 +44,15 @@
         role: ci_setup
 
     - name: Prepare install_yamls make targets
+      when:
+        - cifmw_architecture_scenario is undefined
       tags:
         - bootstrap
```
[the remark this answers is not in the document]
Comment (cjeanner) [reply playbooks/01-bootstrap.yml:50]: we prepare install_yamls when that parameter is *un*defined, yes :). Meaning we're not in a n arch scenario.

File: playbooks/01-bootstrap.yml [hunk 2]
```diff
@@ -44,10 +44,15 @@
...
-      ansible.builtin.import_role:
+      ansible.builtin.include_role:
```
[the remark this answers is not in the document]
Comment (cjeanner) [reply playbooks/01-bootstrap.yml:51]: nope, it's totally related actually: the condition would be applied to all of the imported tasks, while, with include, we just don't include anything.

File: playbooks/01-bootstrap.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (marios) [reply playbooks/01-bootstrap.yml:50]: ack i was confused by the the commit message "Do not initialize install_yamls in architecture scenario"
so conversely, "Do initialize install_yamls when not in arch scenario"  which is what you have in code

**Pull request merged**